sound/soc/sof/intel/hda-ipc.c

```diff
@@ -161,7 +167,7 @@ irqreturn_t hda_dsp_ipc_irq_thread(int irq, void *context)
 					HDA_DSP_REG_HIPCCTL_DONE, 0);
 
 		/* handle immediate reply from DSP core - ignore ROM messages */
-		if (msg != 0x1004000) {
+		if (hda_dsp_ipc_is_sof(msg)) {
```

Comment:
is_sof means not ROM here? the name is weird.

Reply:
Yes, I thought `if (is_sof())` is better than `if (!is_rom())` but better name proposals are always welcome.